packages/capacitor-plugin/src/web.ts

```diff
@@ -3,9 +3,25 @@ import { WebPlugin } from '@capacitor/core';
 import type { CallbackID, ClearWatchOptions, GeolocationPluginPermissions, IGeolocationPlugin, PermissionStatus, Position, PositionOptions, WatchPositionCallback } from './definitions';
 
 export class GeolocationPluginWeb extends WebPlugin implements IGeolocationPlugin {
-  getCurrentPosition(options?: PositionOptions): Promise<Position> {
-    throw new Error('Method not implemented.');
+  async getCurrentPosition(options?: PositionOptions): Promise<Position> {
+    return new Promise((resolve, reject) => {
+      navigator.geolocation.getCurrentPosition(
+        pos => {
+          resolve(pos);
+        },
+        err => {
+          reject(err);
+        },
+        {
+          enableHighAccuracy: false,
+          timeout: 10000,
+          maximumAge: 0,
+          ...options,
```

Comment:
This is how it already is for the current capacitor version, do we know if there's any reason why we don't/didn't use `PositionOptions` here?

Reply:
we actually could do what we do in cordova: the merge of the javascript obejcts

Reply:
Oh you're right, I read it wrong my bad 😆